lib/Mojo/WebSocketProxy/Dispatcher/Config.pm

```diff
@@ -0,0 +1,59 @@
+package Mojo::WebSocketProxy::Dispatcher::Config;
+
+use Mojo::Base -base;
+
+my $actions;
+my $config;
+
```

Comment:
It still has shared `my $actions;` and `my $config`, and they are used in `init`